modules/rtbdemandBidAdapter.js

```diff
@@ -0,0 +1,126 @@
+import * as utils from 'src/utils';
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'rtbdemand';
+const BIDDER_SERVER = 'bidding.rtbdemand.com';
+export const spec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: function(bid) {
+    return !!(bid && bid.params && bid.params.zoneid);
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    return validBidRequests.map(bidRequest => {
+      var server = bidRequest.params.server || BIDDER_SERVER;
+      console.log(JSON.stringify(bidderRequest));
+      console.log(JSON.stringify(bidderRequest.bids[0].sizes[0]));
```

Comment:
Prebid doesn't `console.log` anything unless it's being run in debug mode.

Use `utils.logMessage()` for these instead